test/DynamoCoreWpfTests/DynamoTestUIBase.cs

```diff
@@ -84,6 +89,11 @@ public virtual void Start()
             View.Show();
 
             SynchronizationContext.SetSynchronizationContext(new DispatcherSynchronizationContext());
```

Comment:
What will the current thread be typically in which this test fixture is set up?

Reply:
We run all our tests in a single thread (UI thread).
We set the current syncContext the same way WPF would set when running the application normally (DispatcherSynchronizationContext)

Reply:
It's surprising to me that if the tests are running in the UI thread, we have to do this explicitly. Perhaps this is a subtle difference between a main WPF app and a WPF test.